openapi-core/src/main/java/io/ballerina/openapi/core/generators/common/GeneratorUtils.java

```diff
@@ -396,6 +396,9 @@ public static String escapeIdentifier(String identifier) {
      * @return string with new generated name
      */
     public static String getValidName(String identifier, boolean isSchema) {
+        if (identifier.isBlank()) {
```

Comment:
shall we handle blank scenarios in particular places as well? ex:  if the header is with blank name, it can not be accepted as a header name, therefore we need to provide a user warning or some other way to handle it.

Reply:
Updated